_commands/basics/whoami.md

```diff
@@ -3,22 +3,28 @@
 
 whoami
 --
-TODO: Add documentation for this command by submitting a pull request.
+`whoami` prints the username of the current user when invoked
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ whoami
+mmdarden
 ~~~
 
 <!--more-->
 
-### Useful Options / Examples
-
-#### Example command
-
-##### Break it down
+### Example Command
+`whoami` returns root if you are the system administrator
+~~~ bash
+$ whoami
+root
+~~~
+#### 
+`whoami` has been obsoleted by the `id` command
```

Comment:
The `id` command is another command in the reference page. Can you link to that here? That way people can easily read more about it.